README.md

```diff
@@ -12,10 +12,14 @@ This is where we collect all of the best stuff from the ecosystem and community.
 </p>
 
 ## Table of Contents
+- [Table of Contents](#table-of-contents)
 - [Get Started](#get-started)
+  - [Templates](#templates)
 - [Plugins](#plugins)
 - [Integrations](#integrations)
 - [Apps](#apps)
+  - [Open Source](#open-source)
+  - [Closed Source](#closed-source)
 - [Tutorials](#tutorials)
 - [Articles](#articles)
```

Comment:
looks like this section might be a maintenance burden, lets just remove it. github automatically creates one that should be enough. cc @JonasKruckenberg

Reply:
Ah rip, I must've accidentally saved which made it auto format. I do think these are useful though - In my other PR I added them, but made the heading to `###` which removes the unnecessary `Table of contents`.